package.json

```diff
@@ -33,13 +36,15 @@
     "@types/cheerio": "^0.22.23",
     "@types/json-diff": "^0.5.0",
     "@types/json-schema": "^7.0.6",
+    "@types/lodash": "^4.14.167",
```

Comment:
This is needed as a fix until https://github.com/bcherny/json-schema-to-typescript/pull/362 is landed.